### DISCUSSION TITLE
Example for function assignment

### DISCUSSION THREAD

**Discussion by ETCappy** [microsoft/Web-Dev-For-Beginners#423] [Q&A]
Hello-
I really like this program but examples for the assignments would be SO helpful as a new teacher.  Can you provide an example for this assignment?
Thanks
[https://github.com/microsoft/Web-Dev-For-Beginners/blob/main/2-js-basics/2-functions-methods/assignment.md]

**Comment by jlooper** [comment]
I will dig from our boot camp archive - also I’ll make an answer key soon 👍

**Comment by ETCappy** [comment]
Hi Jen!
Thanks so much for getting me these examples!  Really helpful and I will be
using these as a guide to help the students.
Much appreciated and have a good weekend!

Elizabeth

E.Thomas-Cappello

Newburgh Free Academy-Main

Computer Science Pathway

ECHS Marist/Newburgh Partnership

AWS Educate Cloud Ambassador

<https://twitter.com/ecappellonecsd> <https://www.instagram.com/nfacsmd/>


On Sat, Sep 25, 2021 at 11:27 AM Jen Looper ***@***.***>
wrote:

> I will dig from our boot camp archive - also I’ll make an answer key soon
> 👍
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/Web-Dev-For-Beginners/discussions/423#discussioncomment-1383775>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL35LM7ZSVDBDJ7JJPB2343UDXS7LANCNFSM5ESLBWJA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
This is a staff email account managed by the Newburgh Enlarged City School 
District.  This email and any files transmitted with it are confidential 
and intended solely for the use of the individual or entity to whom they 
are addressed. If you have received this email in error please notify the 
sender.
